kapitan/cli.py

```diff
@@ -286,13 +284,25 @@ def build_parser():
         default=from_dot_kapitan("compile", "yaml-multiline-string-style", "double-quotes"),
         help="set multiline string style to STYLE, default is 'double-quotes'",
     )
-
     compile_parser.add_argument(
         "--yaml-dump-null-as-empty",
         default=from_dot_kapitan("compile", "yaml-dump-null-as-empty", False),
         action="store_true",
         help="dumps all none-type entries as empty, default is dumping as 'null'",
     )
+    compile_parser.add_argument(
+        "--helm-secrets",
```

Comment:
Can I suggest that we rename this into `--helm-refs`? Reason being that not only secrets could be used here but also other non encrypted refs if the user wishes to.

Reply:
Sure. It's done in the latest commit